public/facilityData/zmaSectors.json

```diff
@@ -31,11 +31,6 @@
               [-80.98611111, 28.72777778],
               [-80.98333333, 28.66666667],
               [-80.430556, 27.306111],
-              [-80.3825, 27.133611],
-              [-80.22166667, 27.15],
-              [-80.06666667, 27.15],
-              [-79.95, 27.17083333],
-              [-79.83910896, 27.25185306],
```

Comment:
whoops, I guess this can be removed

Reply:
the commit should be removed? or the data removed within the file should stay removed?